cmake/toolchains/c.windows.common.cmake

```diff
@@ -26,4 +26,7 @@ set(CMAKE_FIND_ROOT_PATH_MODE_PROGRAM NEVER)
 set(CMAKE_FIND_ROOT_PATH_MODE_LIBRARY ONLY)
 set(CMAKE_FIND_ROOT_PATH_MODE_INCLUDE ONLY)
 
-set(Boost_COMPILER "-mgw44")
+list(APPEND CMAKE_C_IMPLICIT_INCLUDE_DIRECTORIES ${CMAKE_FIND_ROOT_PATH}/include)
+list(APPEND CMAKE_CXX_IMPLICIT_INCLUDE_DIRECTORIES ${CMAKE_FIND_ROOT_PATH}/include)
+
+set(Boost_USE_STATIC_RUNTIME ON)
```

Comment:
I've retried the boost build, I could remove the static runtime flag if we want to? I don't know if that makes any difference for our binaries at all?

Reply:
oh and I also could build dynamic boost libraries, but then we have to "ship" them